src/scss/_header.scss

```diff
@@ -40,6 +40,54 @@
             }
         }
     }
+
+    // The nav bar will be pinned on mobile but not on desktop.
+    &.registry {
+        @apply sticky z-50 bg-gray-850;
+        top: -1px;
+
+        @screen md {
+            @apply relative z-auto;
+        }
+
+        &.is-pinned {
+            @apply shadow;
+
+            @screen md {
+                @apply shadow-none;
+            }
+        }
+
+        .logo-nav-button {
+            @apply flex flex-row block cursor-pointer items-center;
+        }
+
+        .logo-nav-menu {
+            @apply w-full bg-gray-850 pt-2 -ml-2 text-white font-display;
+
+            z-index: 100;
```

Comment:
Does a Tailwind `z-50` not work here?

Reply:
Something something tried this for something then forgot to revert it. Now it is and I am once again using TW utilities.